osu.Game.Rulesets.Osu/Mods/OsuModRandom.cs

```diff
@@ -28,7 +28,7 @@ public class OsuModRandom : ModRandom, IApplicableToBeatmap
 
         public override Type[] IncompatibleMods => base.IncompatibleMods.Append(typeof(OsuModTargetPractice)).ToArray();
 
-        [SettingSource("Angle sharpness", "How sharp angles should be", SettingControlType = typeof(SettingsSlider<float>))]
+        [SettingSource("Angle sharpness", "How sharp angles should be", SettingControlType = typeof(AngleSharpnessSlider))]
```

Comment:
I don't get this. The previous code used `SettingControlType` for what looks like pretty much no reason. Then why double down on that and add a special slider implementation here just to change `KeyboardStep`, if it makes the setting slider behave completely differently to *every* other setting slider? Difficulty Adjust, or any other mod, does not do this.

If you ask me, this PR should be a one-liner of:

```suggestion
        [SettingSource("Angle sharpness", "How sharp angles should be")]
```

and that's it. If there's an issue with the fact that will change `KeyboardStep` to be equal to precision (`0.1f`), then it's likely a UX issue with every single setting slider and should be addressed in `SettingSourceExtensions.CreateSettingsControls()` to be consistent everywhere.

Unless there is somehow some special reason why this one specific slider *needs* the keyboard step to be that precisely that I'm not seeing?

Reply:
I didn't see a reason why either. I just did it to fix the issue, since it seemed to need to go by 0.5 by default. I'll add this change in the next commit.